CHANGES.rst

```diff
@@ -13,6 +13,15 @@ Changelog
 
 .. towncrier release notes start
 
+1.1.0 (2020-08-18)
```

Comment:
We want this release to be 1.0.2 I think and then master would be 1.1.0.dev still.